adapter/internal/messaging/azure_listener.go

```diff
@@ -19,32 +19,42 @@
 package messaging
 
 import (
+	"time"
 	"github.com/wso2/product-microgateway/adapter/config"
 	logger "github.com/wso2/product-microgateway/adapter/internal/loggers"
-	msg "github.com/wso2/product-microgateway/adapter/pkg/messaging"
-	servicebus "github.com/Azure/azure-service-bus-go"
 	"github.com/wso2/product-microgateway/adapter/pkg/health"
-	"time"
+	msg "github.com/wso2/product-microgateway/adapter/pkg/messaging"
 )
 
 const (
-	componentName string = "adapter"
-	subscriptionIdleTimeDuration = time.Duration(72 * time.Hour)
+	componentName                              = "adapter"
+	subscriptionIdleTimeDuration               = time.Duration(72 * time.Hour)
+	defaultReconnectRetryCount                 = 60
+	defaultReconnectInterval     time.Duration = 5000
```

Comment:
Shall we add a comment that this in ms

Reply:
According to below comment we can remove this now. Will add a comment in the place we add default values

Reply:
fixed in https://github.com/wso2/product-microgateway/pull/2196/commits/f6486f812ca86d1d6756e920332a21758e1e99c6